src/daemon/index.ts

```diff
@@ -103,21 +96,45 @@ async function traceLSPPerformance(javaExt: vscode.Extension<any>) {
       return;
    }
 
-   lspUsageStats = new LSPUsageStats(javaExtVersion, sampling);
+   // Enable it since redhat.java@1.23.0
+   if (javaExt.exports?.onWillRequestStart) {
+      lspUsageStats = new LSPUsageStats(javaExtVersion, sampling);
+      try {
+         // Load HdrHistogramJS WASM module
+         if (vscode.env.uiKind === vscode.UIKind.Desktop) {
+            hdr.initWebAssemblySync();
```

Comment:
Any specific reason to use the sync version?

Reply:
This is the recommended approach from the library. On server side, we can use initAsync directly, but for browser, use the async.

https://github.com/HdrHistogram/HdrHistogramJS#boosting-performances-with-webassembly